### ISSUE TITLE
include label additions and uploads on activity chart

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#204]
I notice refinements are there too,

perhaps a unified 'activity bar' would help given the number of different types of activity..

I realise that validations will appear after label additions but it's indirect,

it's definitely he case that *annotations* have most value ... that's an argument to retain them being singled out

are there other ways to do this? (maybe just 'be accurate' and show seperate bars for upload, annotation,label,refinement,validation..) or a drop-box? ("show: total activity , annotations, validations,...")

**Comment by bbernhard** [comment]
You are right, the uploads are missing in the activity. I think the restrictions date back to the early days, where it wasn't possible to upload unlabeled donations. As every upload had at least one label it wasn't necessary to track image uploads separately. But with the possibility to upload unlabeled donations, we should probably add the donations to the activity chart as well.

If possible, I would be in favor of the "be accurate", as it gives a nice overview of what's going on at first glance. Not sure however, if we can add another bar to the activity chart, or if it will be too much then...but let's see :)

btw: I recently reworked the [statistics page](https://imagemonkey.io/statistics) a bit. There is still a bit of work needed (e.q: I think it might be a good idea to show tooltips with explanations when hovering over stats), but I _think_ it's a bit better now than before. 

I'll be starting with another round of (UI) improvements soon. I guess that would be a good candidate :)

Thanks for raising this!

**Comment by dobkeratops** [comment]
great to see! .. I try to dip in every now and again regularly even just to keep the app looking used.. doing a quick blast of pure labelling (without annotations) can be the most relaxing part. (I had some other collaborations to put time into so was taking a breather but am trying to find ways to connect between them .. a growable labelled image repository is an awesome resource to have actively maintained)
